crates/hotshot/src/tasks/mod.rs

```diff
@@ -546,9 +543,9 @@ pub async fn add_network_message_and_request_receiver_tasks<
     add_network_message_task(handle, &network);
     add_network_message_task(handle, &network);
 
-    if let Some(request_receiver) = network.spawn_request_receiver_task().await {
+    if let Some(_request_receiver) = network.spawn_request_receiver_task().await {
```

Comment:
i dont think we will need this at all anymore

Reply:
Yes as a follow up task to this we should clean this up.  We should completely remove `spawn_request_receiver_task` and the associated stuff in libp2p, as well as `request_data`

Reply:
i will make a separate task for this